lib/members_page.rb

```diff
@@ -0,0 +1,9 @@
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  field :members do
+    noko.css('.liste2 .liste a').map do |a|
+      MemberPage.new(response: Scraped::Request.new(url: a.attr('href')).response)
```

Comment:
[In other scrapers we've been using a `member_urls` method](https://github.com/search?l=&q=member_urls+user%3Aeverypolitician-scrapers&ref=advsearch&type=Code&utf8=%E2%9C%93) which just returns a list of the `href` attributes. These urls are then used in `scraper.rb` to create `Scraped::Request` and `MemberPage` instances.

The reasons for doing this are twofold. First if someone just wants to get a list of member urls then they have the overhead of every member page being fetched. Second it reduces the number of dependencies `MembersPage` has. Rather than depend on `MemberPage` and `Scraped::Request` it can just return an array of strings.